### DISCUSSION TITLE
Can we have a `--watch` option for `wxt build`

### DISCUSSION THREAD

**Discussion by zouhangwithsweet** [wxt-dev/wxt#483] [Ideas]
Like:
```bash
wxt build --watch
```

**Comment by zouhangwithsweet** [comment]
Or we can have a option to disabled dev server reload page.

**Comment by aklinker1** [comment]
Sure, I can add a watch mode. Would be useful for Firefox MV3 development CC @yunsii

**Reply by yunsii** [reply]
Good idea, but it is too slow if build every entrypoint with single file, https://github.com/wxt-dev/wxt/issues/357 is necessary.

**Comment by rxliuli** [comment]
I also need this feature, because I'm using duckdb (wasm+webworker), and dev mode is completely unusable due to CSP restrictions. I'm not quite sure how to solve this. Does anyone have any ideas?

---

No worries, the direct wxt build speed is fast enough now, and I can combine it with other tools to make it work. For example:

```sh
watchexec -e ts,tsx --project-origin . -- pnpm build
```
